mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -182,6 +183,14 @@ export class ExperimentView extends Component {
     }
   }
 
+  componentDidMount() {
+    if (this.props.location.pathname === "/") {
+      document.title = "MLflow Experiments";
+    } else {
+      document.title = this.props.experiment.name;
```

Comment:
```suggestion
      document.title = this.props.experiment.name + " - Experiment";
```

Wondering if we can make it clearer that the current page is the experiment view, while keeping the experiment name at the start of the title so that it's possible to disambiguate multiple tabs with experiments open.

Other websites like JIRA / GitHub seem to follow a similar pattern (page titles have prefixes that are likely to be unique, followed by additional information). For example, the title of this page is "Make MLflow run and experiment page titles more descriptive by AveshCSingh · Pull Request #2420 · mlflow/mlflow"

Reply:
cc @gioa on this, the context is we're trying to settle on a good scheme for MLflow UI title pages

Reply:
I like it. I've added your suggested in a new commit.